### PR TITLE
Oscillator

### LINKED ISSUE #3159: Create tests using anharmonic oscillator
We need a non-linear equivalent to the poly-case in order to properly test IES.

### PR DESCRIPTION
**Pull request by dafeda**
**Issue**
Resolves #3159

**Approach**
_Short description of the approach_

## Pre review checklist

- [ ] Added appropriate labels

Adding labels helps the maintainers when writing release notes, see sections and the
corresponding labels here: https://github.com/equinor/ert/blob/main/.github/release.yml

### CODE REVIEW

File: test-data/local/oscillator/time_map [hunk 1]
```diff
@@ -0,0 +1 @@
+01/10/2006
```
Comment (berland) [inline test-data/local/oscillator/time_map:1]: Use ISO-date, `2006-10-01`

File: test-data/local/oscillator/observations [hunk 2]
```diff
@@ -0,0 +1,6 @@
+GENERAL_OBSERVATION OSCILLATOR_OBS {
+   DATA       = OSCILLATOR_RES;
+   INDEX_LIST = 0,50,100,150,200,250,300,350,400,450,500,550,600,650,700,750,800,850,900,950,1000,1050,1100,1150,1200,1250,1300,1350,1400,1450,1500,1550,1600,1650,1700,1750,1800,1850,1900,1950,2000,2050,2100,2150,2200,2250,2300,2350,2400,2450;
+   RESTART    = 0;
```
Comment (berland) [inline test-data/local/oscillator/observations:4]: Some beautiful day `TIME_MAP` will disappear. Prior to that day, the `RESTART` keyword in the observation files might be deprecated. Will it work to just insert the date (ISO-8601) here instead?

File: test-data/local/oscillator/coeff_priors [hunk 3]
```diff
@@ -0,0 +1,2 @@
+COEFF_omega UNIFORM 2.5e-2 4.5e-2
+COEFF_lmbda UNIFORM 2.0e-4 4.0e-4
```
Comment (berland) [inline test-data/local/oscillator/coeff_priors:2]: Python workarounds leaking into ERT config?? ;)

Comment (berland) [reply test-data/local/oscillator/coeff_priors:2]: (lmbda vs lambda)

**Comment by oyvindeide** [comment]
Will close this as it is not actively worked on. Feel free to reopen if you pick this up again.

**Comment by dafeda** [comment]
Closing for now, might pick up later though.

**Pull request closed without merging**